packages/components/organisms/f-offers/src/store/types.js

```diff
@@ -6,3 +6,5 @@ export const ACTION_VALID_SAVED_ADDRESS_SEARCH = 'validSavedAddressSearch';
 export const ACTION_VALID_SEARCH = 'validSearch';
 
 export const MUTATION_EMIT_SEARCH_START = 'searchbox-focus';
+
+export const OFFERS_SEARCH_VUEX_MODULE_NAMESPACE = 'offersSearch';
```

Comment:
I'd prefer if the namespace for this was at the beginning to match the other parts of this file, e.g:
```suggestion
export const VUEX_MODULE_NAMESPACE_OFFERS_SEARCH = 'offersSearch';
```

Reply:
agreed good idea